crates/ark/src/variables/variable.rs

```diff
@@ -588,6 +588,28 @@ fn has_children(value: SEXP) -> bool {
     }
 }
 
+fn has_viewer(value: SEXP) -> bool {
+    if !(r_is_data_frame(value) || r_is_matrix(value)) {
+        return false;
+    }
+
+    // We have a data.frame or matrix. Dispatch to the has_viewer method
+    match ArkGenerics::VariableHasViewer.try_dispatch::<bool>(value, vec![]) {
+        Err(err) => {
+            log::error!(
+                "Error from '{}' method: {err}",
+                ArkGenerics::VariableHasViewer.to_string()
+            );
+            // The viewer exists, but failed
+            true
```

Comment:
```suggestion
            false
```

I think I have a slight preference for being safer here and saying no